skiko/src/awtMain/cpp/windows/DCompLibrary.hh

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <dcomp.h>
+
+namespace DCompLibrary {
+    HINSTANCE getLibrary() {
```

Comment:
It's supposed to be file-local so mark it `static` (after moving from the header)

Reply:
Done